norfair/drawing.py

```diff
@@ -1,10 +1,18 @@
+from typing import Optional, Tuple, List
+
 import cv2
 import numpy as np
 
 from .utils import validate_points
 
 
-def draw_points(frame, detections, radius=None, thickness=None, color=None):
+def draw_points(
+    frame: np.array,
+    detections: List,
```

Comment:
Instead of `List` we could be less specific here and use `Sequence `. Also adding `"Detection"` to avoid having trouble with `Detection` being defined in this same file.